stripe/src/main/java/com/stripe/android/StripeApiRepository.kt

```diff
@@ -814,6 +815,20 @@ internal class StripeApiRepository @JvmOverloads internal constructor(
         }
     }
 
+    override suspend fun getCardMetadata(binPrefix: String, options: ApiRequest.Options) = flow {
```

Comment:
We don't need to use `Flow` here - see #2660

Reply:
done